src/ssl.c

```diff
@@ -4813,6 +4815,17 @@ int MatchTrustedPeer(TrustedPeerCert* tp, DecodedCert* cert)
 #endif /* WOLFSSL_TRUST_PEER_CERT */
 
 
+/* return the CA common name that was used for the handshake */
+int getCurrentCAName(char *name)
```

Comment:
This name does not confirm to public API's and must also be in the header with `WOLFSSL_API`. The length of `name` needs to be passed in also to prevent possible overrun.